### ISSUE TITLE
Unicode glyphs not showing in Firefox

### ISSUE THREAD

**Issue by finanalyst** [Raku/doc-website#333]
I am having a problem getting Firefox to display some Unicode glyphs, in particular U+2A75 (eg [numerical equality](https://docs.raku.org/language/operators#infix_==,_infix_%E2%A9%B5) )

The problem is not related to fonts available to the OS, because the char can be seen in a terminal, nor is it confined to docs.raku.org. Here is a screen shot of a web site dedicated to Unicode in Firefox and in Chromium:
![Searching for U+2A75](https://github.com/Raku/doc-website/assets/363676/1cbc66ba-ed8b-441e-897f-43482cb6ead1)

I have tried a number of config options, and trying to get the font-family changed, but nothing yet is successful. I have submitted a help ticket to Mozilla/Firefox.

The issue here is that perhaps in one of the pages we could indicate that when difficulties are encountered with Unicode glyphs, a user should try another browser.

**Comment by finanalyst** [comment]
tl;dr There's a config flag in Firefox to disable some code points. Set to False, and the glyph will be shown properly.

Mozilla came back to me with a reply about disfunctions with Unicode glyphs in some ranges. After Firefox 119, a new privacy key `privacy.fingerprintingProtection` was added.
Setting this to `True` prevents some Unicode ranges to be shown. So setting to `False` allows the glyph to be seen.
However, the key was introduced as a security measure, so setting to False is something to make an informed choice about: I'll not explain here as it's off topic.
